src/SFA.DAS.PSRService.Web/Controllers/QuestionController.cs

```diff
@@ -50,10 +50,31 @@ public IActionResult Index(string id)
 
             if (sectionViewModel.CurrentSection.Questions != null)
                 sectionViewModel.Questions = currentSection.Questions.Select(s => new QuestionViewModel { Answer = s.Answer, Id = s.Id, Optional = s.Optional, Type = s.Type }).ToList();
-
+            sectionViewModel.CurrentSection.ZenDeskLabel = GetZendeskLabel_ByQuestionId(id);
             return View("Index", sectionViewModel);
         }
 
+        private string GetZendeskLabel_ByQuestionId(string id)
```

Comment:
👍

Reply:
I meant does is really belong in the controller? maybe it belongs in the view model and the view calls a method on the view model which does that translation rather than the other way around.

Also GetZendeskLabel_ByQuestionId method wherever its located is just crying out for a comment to explain why its used - I know EFSA is institutionally opposed to adding comments of any kind for whatever reason - but given the oddness of having to do this workaround we could stretch to a comment on this occasion :)